src/domain/usecases/SaveUserStatusUseCase.ts

```diff
@@ -10,11 +9,11 @@ export class SaveUserStatusUseCase implements UseCase {
     public execute(users: User[], options: SaveUserStatusOptions): FutureData<void> {
         try {
             const usersToUpdate = users.map(user => {
-                return User.createNew({ ...user, disabled: options.disabled }).getOrThrow();
+                return options.disabled ? user.disable().getOrThrow() : user.enable().getOrThrow();
             });
             return this.userRepository.save(usersToUpdate).toVoid();
         } catch (error) {
-            return Future.error(i18n.t(`${(error as Error).message}`));
```

Comment:
Same, I guess the catch is needed because of the getOrThrow. It's old code, we can leave it as-is, but we can keep it in mind for future code.